monetio/models/__init__.py

```diff
@@ -12,6 +12,7 @@
     ncep_grib,
     pardump,
     prepchem,
+    raqms,
```

Comment:
Can we get a small test file for RAQMS? Otherwise might want to give it the `_` / `_mm` treatment too

Reply:
Let me check with Maggie

Reply:
@mebruckner I added some tests in a765270. I also made some editorial changes to the reader. Can you check that everything looks fine to you?

Reply:
@mebruckner can you confirm updated code here works for your larger set of files?